dll/opengl/glu32/src/libnurbs/internals/quilt.cc

```diff
@@ -62,7 +62,8 @@ Quilt::deleteMe( Pool& p )
 {
     for( Quiltspec *q=qspec; q != eqspec; q++ ) {
 #if 1
-	if( q->breakpoints) delete[] q->breakpoints;  q->breakpoints = 0;
+	if( q->breakpoints) delete[] q->breakpoints;
+    q->breakpoints = 0;
```

Comment:
Are there real changes here? Or is it just coding style changes?
This is hard to review.
If that's just coding style, I suggest to revert.
Especially since it may be third party synced code.